src/packet/signature.js

```diff
@@ -315,7 +315,9 @@ class SignaturePacket {
     if (!this.issuerKeyID.isNull() && this.issuerKeyVersion < 5) {
       // If the version of [the] key is greater than 4, this subpacket
       // MUST NOT be included in the signature.
-      arr.push(writeSubPacket(sub.issuerKeyID, true, this.issuerKeyID.write()));
+      // XXX: making this critical breaks RPM <=4.16.
+		  // See: https://github.com/ProtonMail/go-crypto/issues/263
```

Comment:
```suggestion
      // Note: making this critical breaks RPM <=4.16.
      // See: https://github.com/ProtonMail/go-crypto/issues/263
```
(the linter complains)